react-regex.js

```diff
@@ -174,5 +174,15 @@ ReactRegex = [
   {
     regex: /\stabindex=/g,
     replace: " tabIndex="
+  },
+  // Template.contentBlock
+  {
+    regex: /({{>\s*Template.contentBlock\s*}})/g,
+    replace: "{this.props.children}"
+  },
+  // @index -- match @index by itself {{@index}} or as a param to helper {{helper @index anotherParam}}
+  {
+    regex: /({{.*@index.*}})/g,
+    replace: "{index}"
```

Comment:
I'm haven't tested this, but it feels like this would replace the whole `{{helper @index anotherParam}}` with {index}, right?